discovery_engine/lib/src/domain/document_manager.dart

```diff
@@ -88,5 +106,16 @@ class DocumentManager {
     }
     activeData.addViewTime(mode, Duration(seconds: sec));
     await activeRepo.update(id, activeData);
+
+    // As we don't have a `DocumentViewMode` on the Rust side at the moment,
+    // we need to decide which value to use or to aggregate all view modes.
+    final sumDuration = activeData.viewTime.values
```

Comment:
Can you ask pink what they want?

Reply:
@bartlomiej-matejczy confirmed it should be sum of all view times.